src/systems/tracked-controls-webxr.js

```diff
@@ -19,7 +19,16 @@ module.exports.System = registerSystem('tracked-controls-webxr', {
   updateControllerList: function () {
     var xrSession = this.el.xrSession;
     var self = this;
-    if (!xrSession) { return; }
+    if (!xrSession) {
+      // Broadcast that we now have zero controllers connected if there is
+      // no session
+      if (this.oldControllersLength > 0) {
+        this.oldControllersLength = 0;
+        this.controllers = [];
+        this.el.emit('controllersupdated', undefined, false);
```

Comment:
Why is this necessary? `controllerdisconnected` should be [fired](https://github.com/aframevr/aframe/blob/master/src/utils/tracked-controls.js#L44)

Reply:
From what I'm seeing, we hit an error which states that we have "Failed to execute 'getPose' on 'XRFrame': XRSpace and XRFrame sessions do not match.", which seems to stem from an old session id provided by the controllers.  This appears to be cached from the system.controllers.